dmutils/content_loader.py

```diff
@@ -368,7 +368,7 @@ def get_question(self, field_name):
     def get_data(self, form_data):
         if self.fields:
             return {
-                key: form_data[key]
+                key: form_data[key] if form_data[key] else None
```

Comment:
This only sends `None` for questions with `fields`. Currently they can only be text fields.